### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by nodejs-crowdin**

### CODE REVIEW

File: content/v10.x/ar-SA/doc/api/addons.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
-# إضافات C++&lrm;
+# C++ Addons
```
Comment (nschonni) [inline content/v10.x/ar-SA/doc/api/addons.md:1]: This appears to be reverting translations

**Comment by alexandrtovmach** [comment]
@nschonni I've updated the export configuration:
image
all of these translations are not proofreaded and approved, so reverted to original

**Comment by alexandrtovmach** [comment]
I agree that looks weird to untranslate content, but I'm still trying to figure out optimal config for projects.

**Comment by nschonni** [comment]
I think the "skip untranslated files" should probably be enabled then. Fallback to English is already handled by the server

**Comment by alexandrtovmach** [comment]
Closed in favor of changing export settings

**Pull request closed without merging**